tests/unit/layouts/ModelizerTextLayout.spec.js

```diff
@@ -110,6 +112,61 @@ describe('Test component: ModelizerTextLayout', () => {
         },
       });
     });
+
+    it('should use query path if there are no models', async () => {
+      getPlugins.mockImplementation(() => ([{
+        isParsable: () => true,
+        getModels: () => ([]),
+        data: {
+          name: 'test',
+        },
+      }]));
+
+      await wrapper.vm.onSelectFileTab('plugin/fileName');
+
+      expect(useRouterPush).toHaveBeenCalledTimes(1);
+      expect(useRouterPush).toHaveBeenCalledWith({
+        name: 'Text',
+        params: {
+          projectName: wrapper.vm.projectName,
+        },
+        query: {
+          path: 'pluginName/modelName',
+          plugin: 'test',
+        },
+      });
+    });
+
+    it('should use query plugin if unparsable file is selected', async () => {
+      getPlugins.mockImplementation(() => ([{
+        isParsable: () => false,
+        getModels: () => ([]),
+        data: {
+          name: 'test',
+        },
+      }]));
+      getPluginByName.mockImplementation(() => ({
+        isParsable: () => false,
+        getModels: () => ([]),
+        data: {
+          name: 'test2',
```

Comment:
replace `test2` by `wrapper.vm.query.plugin` to prove the test

Reply:
done